lib/Twig/Loader/Filesystem.php

```diff
@@ -302,4 +302,34 @@ private function isAbsolutePath($file)
             || null !== parse_url($file, PHP_URL_SCHEME)
         ;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getTemplateFreshness(Twig_CacheInterface $cache)
+    {
+        $cacheKey = $cache->generateKey(
+            md5(serialize($this->cache)),
+            get_class()
+        );
+
+        ob_start();
+        $cache->load($cacheKey);
```

Comment:
@mmicael1 The important point @stof is conveying is:

> Twig_CacheInterface is meant to cache class definition for compiled templates [and] **reusing it for a totally different caching need is a bad idea**, as the implementation may not be a generic cache.

I fail to see how your response relates to the important point made.